.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: ruby
```

Comment:
This file is merged from root repo, I would probably remove it for now and setup travis integration separately, it'll need some tweaks for sure.